src/wallet/userWallet.ts

```diff
@@ -242,4 +243,13 @@ export class UserWallet {
             crypto: cryptoSection,
         };
     }
+
+    save(path: PathLike, addressHrp: string = LibraryConfig.DefaultAddressHrp): void {
```

Comment:
Seems fine :crossed_fingers: Alternatively, we could have left the caller to be responsible with providing a good & safe path (either relative or absolute, without additional handling)?

:thought_balloon: